models/demos/tg/llama3_70b/tt/llama_decoder_galaxy.py

```diff
@@ -254,16 +253,16 @@ def decode_forward(
         )
         attn_outs.deallocate(True)
 
-        output_interleaved = ttnn.to_memory_config(output, memory_config=ttnn.DRAM_MEMORY_CONFIG)
-        ffn_norm_out = self.tt_distributed_rmsnorm(
-            output_interleaved,
+        ffn_norm_out = tt_sharded_distributed_rmsnorm(
+            self.mesh_device,
+            output,
             epsilon=self.norm_eps,
             gamma=self.ffn_norm_sharded,
         )
 
         ffn_norm_out = ttnn.to_memory_config(ffn_norm_out, memory_config=self.MLP_ACT_MEMCFG)
         ffn_out = self.mlp(ffn_norm_out, mode="decode")
-
+        ffn_norm_out.deallocate(True)
```

Comment:
Why didn't we need this before? If we do need it for self.tt_distributed_rmsnorm too, can you add it to prefill?

Reply:
I suppose no need because activations are in dram anyway for prefill